src/components/Icons/Icon.tsx

```diff
@@ -55,51 +55,56 @@ export interface IconProps {
     /** Modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
     modifiers?: string[];
     /** Name of the icon */
-    name: IconNames | LogoNames;
+    name?: IconNames | LogoNames;
```

Comment:
What happens when you pass neither a name nor a child?

Reply:
Good point. I added a warning